pkg/generate/app/imageref.go

```diff
@@ -140,6 +140,10 @@ type ImageRef struct {
 	OutputImage     bool
 	Insecure        bool
 	HasEmptyDir     bool
+	// For keeping track of if this image was retrieved from a Dockerfile,
```

Comment:
proper godoc starts w/ the object name, so "FromDockerfile keeps track of....."

same for the other one above.